### ISSUE TITLE
BackBlaze Plugin Playback Not Working

### ISSUE THREAD

**Issue by bbyrdhouse** [WWBN/AVideo#4400]
**Describe the question**
I turned on the Backblaze plugin today. Entered all the proper credentials and "tested" the plugin to see if it would upload the .txt file. It did.

So I cleared the cache and uploaded a small video to test to make sure the video would work properly.
After uploading, I can navigate to the page where the video is but it will not play. The DevTools say 404.
So I go back to Backblaze to look in the bucket and the video never made it to the bucket even though the txt file did.

Do I need to turn something off or on?

<img width="717" alt="Screen Shot 2021-01-19 at 6 52 08 AM" src="https://user-images.githubusercontent.com/7376298/105036946-dc84a300-5a22-11eb-8493-696756827fa3.png">

<img width="600" alt="bb" src="https://user-images.githubusercontent.com/7376298/105037173-35ecd200-5a23-11eb-8286-153f6efaeb25.png">

**Comment by bbyrdhouse** [comment]
NOTE: When I do a livestream test, the video gets recorded and is playable. The files get uploaded to the back blaze bucket ... 

I am going to upload another video to see if it was something like not enough time having passed from the creation of a new bucket to be ing able to store files ...

**Comment by bbyrdhouse** [comment]
UPDATE:
So, I think the issue was that I had created a bucket and then tried to use it too soon. I have not tested a livestream recording as well as an mo4 upload and they both get sent to Backblaze.

One odd think I noticed though is that after uploading an mp4, it does not show up on the home page until after I clear the cache. My question is, is that just local to me or will nobody be able to see the upload until I clear the cache?

If it is the latter, is there a way to set up a cron job to clear the cache like every 5 minutes or so? or better yet, to automatically clear the cache after every upload.
